### DISCUSSION TITLE
There is a way to handle the webhooks synchronously?

### DISCUSSION THREAD

**Discussion by fain182** [spatie/laravel-stripe-webhooks#170] [Q&A]
I am trying to avoid any delay in working on the webhooks, I know I can avoid the `ShouldQueue` method to make my jobs synchronous, but I see that the `Spatie\StripeWebhooks\ProcessStripeWebhookJob` is using anyway the queue (and I suppose it calls my custom job).

There is some configuration that I can use to avoid using the queue at all?

**Comment by benlwong** [comment]
I haven't tried it, but can you just change the following in config/stripe-webhooks.php to point to a new connection and set that connection to sync? In my local environment, I set QUEUE_CONNECTION=sync, and all the jobs execute immediately. In your case, you probably want to create a new connection and set it to sync just for this use case.

```
/*
 * Specify a connection and or a queue to process the webhooks
 */
'connection' => env('STRIPE_WEBHOOK_CONNECTION'),
'queue' => env('STRIPE_WEBHOOK_QUEUE'),
```
